data/test_stories/stories_e2e_conflicting_1.yml

```diff
@@ -0,0 +1,13 @@
+version: "2.0"
```

Comment:
optional: I'd prefer to have these test files inline as they are closer to the actual test and I don't have to jump to the test file to understand the test

Reply:
Done.